source/configure/plugins-configuration-settings.rst

```diff
@@ -505,10 +505,34 @@ Enable plugin
 | - **false**: Disables the Calls plugin on your Mattermost workspace. | - ``config.json`` setting:                |
 +----------------------------------------------------------------------+-------------------------------------------+
 
+.. config:setting:: plugins-callsrtcserveraddress
+  :displayname: RTC server port (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson:
```

Comment:
If this config setting doesn't have an equivalent ``config.json`` setting, please populate the ``config.json`` and ``environment`` fields here as ``N/A``